src/codegeneration/ExplodeExpressionTransformer.js

```diff
@@ -750,6 +752,43 @@ export class ExplodeExpressionTransformer extends ParseTreeTransformer {
     return createCommaExpression(expressions);
   }
 
+  transformFunctionExpression(tree) {
+    // function () {}
+    // =>
+    // $0 = function () {}, $0
+    return this.createCommaExpressionBuilder_().build(tree);
+  }
+
+  transformArrowFunctionExpression(tree) {
+    // () => {}
+    // =>
+    // $0 = () => {}, $0
+    return this.createCommaExpressionBuilder_().build(tree);
```

Comment:
Here it looks like `ExplodeExpressionTransformer` does not descend into the arrow function body and I don't see a change involving function call so I don't understand how this can fix #1934.